client/history/connections.go

```diff
@@ -108,3 +109,7 @@ func (c *connectionPoolImpl) getAllClientConns() []clientConnection {
 
 	return clientConns
 }
+
+func (c *connectionPoolImpl) resetConnectBackoff(cc clientConnection) {
+	cc.grpcConn.ResetConnectBackoff()
+}
```

Comment:
Not sure if we need this helper. Looks a little awkward since it is one statement with the same name

Reply:
I moved it out like this (via an interface) so the caching redirector tests can verify that its called. The basic redirector doesn't use it, since I didn't want to introduce any behavior changes to it.